src/NHSD.BuyingCatalogue.Ordering.Services/NHSD.BuyingCatalogue.Ordering.Services.csproj

```diff
@@ -0,0 +1,18 @@
+<Project Sdk="Microsoft.NET.Sdk">
+
+  <PropertyGroup>
+    <TargetFramework>net5.0</TargetFramework>
+    <GeneratePackageOnBuild>false</GeneratePackageOnBuild>
+  </PropertyGroup>
+
+  <PropertyGroup Condition="'$(Configuration)|$(Platform)'=='Debug|AnyCPU'">
+    <DocumentationFile>D:\Project\GPITFutures\API\13236_ORDAPI\BuyingCatalogueOrdering\NHSD.BuyingCatalogue.Ordering.Services\NHSD.BuyingCatalogue.Ordering.Services.xml</DocumentationFile>
```

Comment:
Can you replace this element with the below element?

```xml
<GenerateDocumentationFile>true</GenerateDocumentationFile>
```

Reply:
Added `GenerateDocumentationFile` element